CMakeLists.txt

```diff
@@ -597,6 +597,10 @@ set(Boost_USE_MULTITHREADED ON)
 find_package(Boost 1.61 COMPONENTS ${BOOST_COMPONENTS} REQUIRED)
 include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 include_directories(SYSTEM ${PROJECT_BINARY_DIR}/include)
+if(FREEBSD)
+  list(REMOVE_ITEM Boost_LIBRARIES "/usr/local/lib/libboost_python.so")
```

Comment:
instead of making a surgery on `Boost_LIBRARIES`, i'd recommend only link the necessary boost libs. i will prepare another PR later on.